### PR TITLE
test: Add test for advisory_get_*_from_text

### PR DESCRIPTION
**Pull request by jelly**
Add two tests for the advisory_get_workaround_from_text and
advisory_get_impact_from_text.

### CODE REVIEW

File: test/test_advisory.py [hunk 1]
```diff
@@ -158,3 +158,26 @@ def test_reporter_cant_edit_advisory(db, client):
...
     assert 1 == advisory_count()
+
+
+def test_advisory_get_impact_from_text(db, client):
+    impact = advisory_get_impact_from_text(DEFAULT_ADVISORY_CONTENT)
+    assert 'Robots will take over' in impact
+    assert 'Impact' not in impact
+    assert 'References' not in impact
+
+    assert advisory_get_impact_from_text('test') == None
```
Comment (anthraxx) [inline test/test_advisory.py]: this tests a completely own thing, this should be moved to a own test case and the method names describing the difference

File: test/test_advisory.py [hunk 2]
```diff
@@ -158,3 +158,26 @@ def test_reporter_cant_edit_advisory(db, client):
...
+
+
+def test_advisory_get_workaround_from_text(db, client):
+    content='\nWorkaround\n==========\n\nUpdate your machine\n\nDescription\n'
+    impact = advisory_get_workaround_from_text(DEFAULT_ADVISORY_CONTENT)
+    assert 'Update your machine' in impact
+    assert 'Workaround' not in impact
+    assert 'Description' not in impact
+
+    assert advisory_get_workaround_from_text('test') == None
```
Comment (anthraxx) [inline test/test_advisory.py]: same here

File: test/test_advisory.py [hunk 3]
```diff
@@ -158,3 +158,26 @@ def test_reporter_cant_edit_advisory(db, client):
...
+
+    content='\nWorkaround\n==========\n\nNone.\n\nDescription\n'
+    impact = advisory_get_workaround_from_text(content)
+    assert advisory_get_workaround_from_text('test') == None
```
Comment (anthraxx) [inline test/test_advisory.py]: what are those lines doing? content and impact is not used anymore and the assert is identical to the previous one

Comment (jelly) [reply test/test_advisory.py]: Hrrmpfft indeed, that looks wrong. But the two lines should test the case where there is no workaround:
```
    if 'None.' == workaround: 
        return None 
```

**Pull request merged**